LICENSES_DEPENDENCIES

```diff
@@ -1809,58 +1810,8321 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+----                          | -------  | --------
+adler32                       | 1.0.2    | BSD-3-Clause
+ansi_term                     | 0.11.0   | MIT
+atty                          | 0.2.3    | MIT
+bitflags                      | 0.7.0    | Apache-2.0, MIT
+bitflags                      | 1.0.0    | Apache-2.0, MIT
+byteorder                     | 1.2.3    | MIT, Unlicense
+clap                          | 2.32.0   | MIT
+coco                          | 0.1.1    | Apache-2.0, MIT
+color_quant                   | 1.0.0    | MIT
+deflate                       | 0.7.17   | Apache-2.0, MIT
+either                        | 1.3.0    | Apache-2.0, MIT
+fuchsia-zircon                | 0.2.1    | BSD-3-Clause
+fuchsia-zircon-sys            | 0.2.0    | BSD-3-Clause
+gif                           | 0.10.0   | Apache-2.0, MIT
+image                         | 0.21.0   | MIT
+inflate                       | 0.4.3    | MIT
+jpeg-decoder                  | 0.1.13   | Apache-2.0, MIT
+kernel32-sys                  | 0.2.2    | MIT
+lazy_static                   | 0.2.9    | Apache-2.0, MIT
+libc                          | 0.2.33   | Apache-2.0, MIT
+lzw                           | 0.10.0   | Apache-2.0, MIT
+num-derive                    | 0.2.2    | Apache-2.0, MIT
+num-integer                   | 0.1.39   | Apache-2.0, MIT
+num-iter                      | 0.1.34   | Apache-2.0, MIT
+num-rational                  | 0.2.1    | Apache-2.0, MIT
+num-traits                    | 0.1.40   | Apache-2.0, MIT
+num-traits                    | 0.2.5    | Apache-2.0, MIT
+num_cpus                      | 1.7.0    | Apache-2.0, MIT
+png                           | 0.14.0   | Apache-2.0, MIT
+proc-macro2                   | 0.4.9    | Apache-2.0, MIT
+quote                         | 0.6.3    | Apache-2.0, MIT
+rand                          | 0.3.18   | Apache-2.0, MIT
+rayon                         | 0.8.2    | Apache-2.0, MIT
+rayon-core                    | 1.3.0    | Apache-2.0, MIT
+redox_syscall                 | 0.1.31   | MIT
+redox_termios                 | 0.1.1    | MIT
+safe-transmute                | 0.10.1   | MIT
+scoped_threadpool             | 0.1.8    | MIT
+scopeguard                    | 0.3.3    | Apache-2.0, MIT
+strsim                        | 0.7.0    | MIT
+syn                           | 0.14.4   | Apache-2.0, MIT
+termion                       | 1.5.1    | MIT
+textwrap                      | 0.10.0   | MIT
+tiff                          | 0.2.1    | MIT
+unicode-width                 | 0.1.4    | Apache-2.0, MIT
+unicode-xid                   | 0.1.0    | Apache-2.0, MIT
+vec_map                       | 0.8.0    | Apache-2.0, MIT
+winapi                        | 0.2.8    | MIT
+winapi                        | 0.3.5    | Apache-2.0, MIT
+winapi-build                  | 0.1.1    | MIT
+winapi-i686-pc-windows-gnu    | 0.4.0    | Apache-2.0, MIT
+winapi-x86_64-pc-windows-gnu  | 0.4.0    | Apache-2.0, MIT
+
+-----BEGIN adler32 1.0.2 LICENSES-----
```

Comment:
adler II

Reply:
But different versions ...; unsure why they are not both listed in the table at the top ...

Reply:
issue in #111 
resolved by https://github.com/foresterre/combostew/pull/20, #114